msi/release.sh

```diff
@@ -1,34 +1,119 @@
 #!/bin/sh
 
-# Build Git for Windows' .msi files
+# Build Git for Windows .msi file
```

Comment:
I think the `'` is still needed.

Oh, and as we are in the `msi/release.sh` file, would you mind prefixing the commit subject with `msi: `? That would make it easier for me when I have to read the commit history later.